chainer/backends/cuda.py

```diff
@@ -28,6 +28,11 @@
 operations.
 """
 
+from typing import Any  # NOQA
```

Comment:
I saw importing `typing` types on top of files in some projects but I'll fix them it is not conventional as you said.

The type level import is also used in [optuna](https://github.com/pfnet/optuna/blob/master/optuna/cli.py#L12-L16). I prefer to use it because writing like `typing.Optional[...]` in signatures is a bit verbose. Any concerns about it?

https://github.com/pfnet/optuna/blob/master/optuna/cli.py#L12-L16